app-doc/cppman/cppman-0.5.4.ebuild

```diff
@@ -29,5 +29,8 @@ src_prepare() {
 	default
 
 	# Don't allow setup.py to install documentation directly
-	sed -i '\:share/doc/cppman:d' setup.py || die "sed failed"
+	sed -i '\:share/doc/cppman:d' setup.py || die
+
+	# Fix zsh completion directory
+	sed -i 's:share/zsh/vendor-completions/:share/zsh/site-functions/:' setup.py || die
```

Comment:
I believe this requires a revision bump

Reply:
You're correct, I didn't forsee the version bump so forgot to do it :+1:

I'll fix both of these things tonight, might take a bit to get home.